Formula/gtk-doc.rb

```diff
@@ -13,24 +13,34 @@ class GtkDoc < Formula
   end
 
   depends_on "pkg-config" => :build
-  depends_on "gnome-doc-utils" => :build
   depends_on "itstool" => :build
   depends_on "gettext"
-  depends_on "glib"
   depends_on "docbook"
   depends_on "docbook-xsl"
   depends_on "libxml2"
   depends_on :perl => "5.18" if MacOS.version <= :mavericks
```

Comment:
I am actually not sure why there is a Perl dependency there at all. There doesn't appear to be any Perl code in the repository